src/tailwind.css

```diff
@@ -26,3 +26,19 @@ h6 {
 .sl-markdown-content code:not(:where(.not-content *)) {
     color: var(--sl-color-gray-1);
 }
+
+.sl-markdown-content h2::before {
+    content: "## ";
+    opacity: 0.5;
+    font-weight: normal;
+}
+.sl-markdown-content h3::before {
+    content: "### ";
+    opacity: 0.5;
+    font-weight: normal;
+}
+.sl-markdown-content h4::before {
+    content: "#### ";
+    opacity: 0.5;
+    font-weight: normal;
+}
```

Comment:
I think this is what's causing those # to show up.

Reply:
Yes, it is intentional. See [`0ea697c` (#58)](https://github.com/blue-build/website/pull/58/commits/0ea697c8001eac8ddb707334c298bf8b1fa08570)

I've grown to like the look. It was added because without it I felt like nested objects in generated docs (like those on the default-flatpaks reference page) were unreadable. Since they're added as ::before elements they should be screenreader friendly.

I could remove them if it looks better without :woman_shrugging:

Reply:
Oh, didn't realize. I'm fine with keeping them.